demo/README.md

```diff
@@ -0,0 +1,61 @@
+### Prep
+
+make quick-release
+local-controlplane-up
+
+### Sample Controller Demo
+
+cd staging/src/k8s.io/sample-controller
```

Comment:
this demo folder should be better in other place, maybe in `staging/src/k8s.io/sample-controller` ? bot not in the root folder

Reply:
thanks, this will be dropped before we merge.